build.gradle

```diff
@@ -277,8 +277,8 @@ dependencies {
     implementation 'org.jooq:jool-java-8:0.9.14'
     implementation 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.6'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.7-CCD-3694_CCD-3836'
```

Comment:
Should be updated with proper release number once hmcts/ccd-test-definitions#57 is released.

Reply:
Updated see https://github.com/hmcts/ccd-data-store-api/pull/2105#discussion_r1029521311